libs/utils/utils.js

```diff
@@ -483,15 +483,23 @@ export function decorateImageLinks(el) {
   const images = el.querySelectorAll('img[alt*="|"]');
   if (!images.length) return;
   [...images].forEach((img) => {
-    const [source, alt] = img.alt.split('|');
+    const [source, alt, playBtn] = img.alt.split('|');
```

Comment:
```suggestion
    const [source, alt, icon] = img.alt.split('|');
```

I would avoid being overly prescriptive here about it being a play button. If anything, all it is right now is an icon. You don't know what it is yet.

Reply:
@auniverseaway I have made the suggested changes.